src/components/form/TextInput/TextInput.js

```diff
@@ -0,0 +1,86 @@
+import React, { useState, forwardRef } from 'react';
+import { View, TextInput as RNTextInput } from 'react-native';
+import { bool, func, shape, string } from 'prop-types';
+
+import Text from 'components/common/Text';
+
+import { inputA11yHint } from '../../../utils/accessibility';
```

Comment:
this repo has folder aliases configured. This can be `'utils/accessibility'`

Reply:
Good catch